setup/setup.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
What was wrong with it?

Reply:
IDK, but it caused errors

Reply:
I think it was probably just line 3 maybe since you had added `/..` but then not removed it from the `pip3` command maybe...

Reply:
Or was there more errors too?

Reply:
Wait... I forgot to change the pip3 line. It works now!